config/proto_conversions.go

```diff
@@ -905,3 +907,11 @@ func PackageTypeToProto(t PackageType) (*packagespb.PackageType, error) {
 		return packagespb.PackageType_PACKAGE_TYPE_UNSPECIFIED.Enum(), errors.Errorf("unknown package type %q", t)
 	}
 }
+
+func logAnyFragmentOverwriteErrors(logger logging.Logger, overwriteFragmentStatus []*pb.AppValidationStatus) {
+	if overwriteFragmentStatus != nil {
+		for _, status := range overwriteFragmentStatus {
+			logger.Errorw("error in overwriting fragment: ", "error", status.GetError())
```

Comment:
```suggestion
			logger.Errorw("error in overwriting fragment", "error", status.GetError())
```
nit: no need for semicolon